web_src/less/_repository.less

```diff
@@ -841,6 +841,10 @@
                     }
                 }
 
+                &:target > .content {
+                    box-shadow: 0 0 10px #8c8c8c;
```

Comment:
Would rather prefer styling like Github, e.g. colored border color and maybe a (very subtle) box shadow, not a heavy one like you have.

Reply:
@silverwind can someone improve this later? It will be hard for this (comment div) for me like github style?

Reply:
I guess I can have a look later.